lib/cpp/src/thrift/transport/TServerSocket.cpp

```diff
@@ -16,7 +16,9 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
+#if defined(_MSC_VER) && (_MSC_VER < 1900)
+#define snprintf _snprintf
```

Comment:
Thank you first, vs2013 has no  snprintf function,but vs2015 has  snprintf function, i see the code of THRIFT_SNPRINTF in PlatformSocket.h,  use the  THRIFT_SNPRINTF replace  snprintf in TServerSocket.cpp will occurr link errors when compiling under vs2015. use my patch code will compile sucessfully under vs2013 and vs2015 at the same time.